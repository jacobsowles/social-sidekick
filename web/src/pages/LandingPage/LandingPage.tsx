import React from 'react';

import ContentBox from '@components/ContentBox';
import PageHeader from '@components/PageHeader';
import './LandingPage.scss';

const LandingPage: React.FunctionComponent = () => {
  return (
    <div className="landing-page">
      <ContentBox>
        <PageHeader title="Landing Page" subtitle="Page subtitle" />
        <p>Page content</p>
      </ContentBox>
    </div>
  );
};

export default LandingPage;
