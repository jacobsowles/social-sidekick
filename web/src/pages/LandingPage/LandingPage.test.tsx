import React from 'react';

import { init, shallow, shallowTopLevelElement, toJson } from '@tests/component-test-base';
import LandingPage from '@pages/LandingPage';

describe('LandingPage', () => {
  init(() => <LandingPage />, 'div');

  it('should render the component without crashing', () => {
    toJson(shallow()).should.matchSnapshot();
  });

  it('should only include the default class name', () => {
    shallowTopLevelElement()
      .prop('className')
      .should.equal('landing-page');
  });
});
