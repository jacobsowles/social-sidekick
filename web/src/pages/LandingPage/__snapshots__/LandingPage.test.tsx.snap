// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should render the component without crashing 1`] = `
<div
  className="landing-page"
>
  <ContentBox>
    <PageHeader
      title="All of your profiles in one place"
    />
    <p>
      New job? Congratulations! But now you're faced with the laborious task of updating your employer on all of your social networks.
    </p>
    <p>
      Looking to revamp your personal brand? Time to update all of your bios.
    </p>
    <p>
      Life changes fast and often. No more blocking off half an hour every time you need to update your profiles. No more digging through menus trying to remember where the settings are. No more forgetting to update certain profiles. Just sign up, connect your accounts, and reclaim your time with Social Sidekick.
    </p>
  </ContentBox>
</div>
`;
